Gopkg.toml

```diff
@@ -23,26 +23,36 @@
 
 [[constraint]]
   name = "github.com/Sirupsen/logrus"
+  version = "1.0.4"
 
 [[constraint]]
   name = "github.com/nlopes/slack"
+  version = "0.1.0"
 
 [[constraint]]
   name = "github.com/spf13/cobra"
+  version = "0.0.1"
 
 [[constraint]]
   name = "github.com/spf13/viper"
+  version = "1.0.0"
 
 [[constraint]]
   name = "github.com/tbruyelle/hipchat-go"
   version = "1.2.0"
 
 [[constraint]]
+  branch = "v2"
   name = "gopkg.in/yaml.v2"
 
 [[constraint]]
+  branch = "master"
+  name = "k8s.io/api"
+
+[[constraint]]
+  branch = "master"
```

Comment:
To be safe, please use specific revisions of `k8s.io/api` and `k8s.io/apimachinery` stuck to client-go v6.0 instead of master. Specifically:

k8s.io/api: https://github.com/kubernetes/client-go/blob/v6.0.0/Godeps/Godeps.json#L375

k8s.io/apimachinery: https://github.com/kubernetes/client-go/blob/v6.0.0/Godeps/Godeps.json#L491